.github/workflows/clear-cache.yaml

```diff
@@ -8,21 +8,26 @@ jobs:
   clear-cache:
     runs-on: ubuntu-latest
     steps:
-      - name: Clear cache
-        uses: actions/github-script@v6
+      - name: Harden Runner
+        uses: step-security/harden-runner@ebacdc22ef6c2cfb85ee5ded8f2e640f4c776dd5 # v2.0.0
         with:
-          script: |
-            console.log("About to clear")
-            const caches = await github.rest.actions.getActionsCacheList({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
```

Comment:
Seems like you could just have increased this with the prop `per_page`, ref https://octokit.github.io/rest.js/v19#actions-get-actions-cache-list